src/main/java/me/figo/FigoApi.java

```diff
@@ -187,19 +188,19 @@ protected void setupTrustManager(HttpURLConnection connection, X509TrustManager
         if (code >= 200 && code < 300) {
             return handleResponse(connection.getInputStream(), typeOfT);
         } else {
-            FigoException.ErrorResponse errorResponse = handleResponse(connection.getErrorStream(), FigoException.ErrorResponse.class);
+            ErrorResponse errorResponse = handleResponse(connection.getErrorStream(), ErrorResponse.class);
 			logError(errorResponse, connection);
-			throw new FigoException(errorResponse);
+			throw new FigoApiException(errorResponse);
         }
     }
 
-	private void logError(FigoException.ErrorResponse errorResponse, HttpURLConnection connection) {
+	private void logError(ErrorResponse errorResponse, HttpURLConnection connection) {
```

Comment:
I would like to remove this log, but I'm  not sure why it was added in the first place. My current workaround is that this logger is blocked by my configuration.